Intermittent failure of test `CxfWsTrustTest.programmaticStsNotSet`

### Bug description

```
2023-06-14T03:36:28.4259208Z Caused by: java.net.ConnectException: ConnectException invoking http://localhost:8081/jaxws-samples-wsse-policy-trust-sts/SecurityTokenService: Connection refused
```

Note the URL has port `8081`. Which probably  comes from here:

https://github.com/apache/camel-quarkus/blob/main/integration-test-groups/cxf-soap/cxf-soap-ws-trust/src/main/resources/AsymmetricSAML2Policy.xml#L50-L52

The resulting exception is thrown from `AbstractSTSClient`:

```
trust}sayHello has thrown exception, unwinding now: org.apache.cxf.ws.security.trust.TrustException: Exception when trying to retrieve/process a WSDL via WS-MEX
2023-06-14T03:47:00.2152933Z 	at org.apache.cxf.ws.security.trust.AbstractSTSClient.configureViaEPR(AbstractSTSClient.java:612)
2023-06-14T03:47:00.2153531Z 	at org.apache.cxf.ws.security.trust.STSUtils.getClientWithIssuer(STSUtils.java:166)
2023-06-14T03:47:00.2154101Z 	at org.apache.cxf.ws.security.trust.STSTokenRetriever.getToken(STSTokenRetriever.java:68)
2023-06-14T03:47:00.2155000Z 	at org.apache.cxf.ws.security.policy.interceptors.IssuedTokenInterceptorProvider$IssuedTokenOutInterceptor.handleMessage(IssuedTokenInterceptorProvider.java:147)
2023-06-14T03:47:00.2155767Z 	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:307)
2023-06-14T03:47:00.2156286Z 	at org.apache.cxf.endpoint.ClientImpl.doInvoke(ClientImpl.java:528)
2023-06-14T03:47:00.2156730Z 	at org.apache.cxf.endpoint.ClientImpl.invoke(ClientImpl.java:439)
2023-06-14T03:47:00.2157149Z 	at org.apache.cxf.endpoint.ClientImpl.invoke(ClientImpl.java:354)
2023-06-14T03:47:00.2157580Z 	at org.apache.cxf.endpoint.ClientImpl.invoke(ClientImpl.java:312)
2023-06-14T03:47:00.2158031Z 	at org.apache.cxf.frontend.ClientProxy.invokeSync(ClientProxy.java:96)
2023-06-14T03:47:00.2158506Z 	at org.apache.cxf.jaxws.JaxWsClientProxy.invoke(JaxWsClientProxy.java:140)
2023-06-14T03:47:00.2158915Z 	at jdk.proxy2/jdk.proxy2.$Proxy102.sayHello(Unknown Source)
```

What's interesting is that test only fails some of the time. Meaning that the exception is probably being swallowed or ignored when the test passes.

--- a/poms/build-parent-it/pom.xml
+++ b/poms/build-parent-it/pom.xml
@@ -84,16 +84,6 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-failsafe-plugin</artifactId>
-                    <executions>
-                        <execution>
-                            <configuration>
-                                <systemProperties>
-                                    <quarkus.http.test-port>${test.http.port.native}</quarkus.http.test-port>
-                                    <quarkus.http.test-ssl-port>${test.https.port.native}</quarkus.http.test-ssl-port>
-                                </systemProperties>
-                            </configuration>
-                        </execution>
-                    </executions>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.camel.maven</groupId>
@@ -138,28 +128,6 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>build-helper-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>reserve-network-port</id>
-                                <goals>
-                                    <goal>reserve-network-port</goal>
-                                </goals>
-                                <phase>process-resources</phase>
-                                <configuration>
-                                    <portNames>
-                                        <portName>test.http.port.jvm</portName>
-                                        <portName>test.https.port.jvm</portName>
-                                        <portName>test.http.port.native</portName>
-                                        <portName>test.https.port.native</portName>
-                                    </portNames>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-
-                    <plugin>
                         <groupId>${quarkus.platform.group-id}</groupId>
                         <artifactId>quarkus-maven-plugin</artifactId>
                         <executions>
@@ -175,20 +143,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>default-test</id>
-                                <phase>test</phase>
-                                <configuration>
-                                    <systemProperties>
-                                        <quarkus.http.test-port>${test.http.port.jvm}</quarkus.http.test-port>
-                                        <quarkus.http.test-ssl-port>${test.https.port.jvm}</quarkus.http.test-ssl-port>
-                                    </systemProperties>
-                                </configuration>
-                            </execution>
-                        </executions>
                     </plugin>
-
                 </plugins>
             </build>
         </profile>

